src/SFA.DAS.Admin.Aan.Web/Controllers/ManageEvent/NetworkEventTypeController.cs

```diff
@@ -73,15 +71,15 @@ private async Task<CreateEventTypeViewModel> GetViewModel(CreateEventSessionMode
 
         var regionsWithNational = regionDropdowns.ToList();
         regionsWithNational.Add(new RegionSelection("National", 0));
-        return new CreateEventTypeViewModel
+        return new EventTypeViewModel
         {
             EventTitle = sessionModel?.EventTitle,
             EventTypeId = sessionModel?.EventTypeId,
             EventRegionId = sessionModel?.EventRegionId,
```

Comment:
sessionModel is not nullable, so why all these null propagators?

Reply:
these are leftovers from the initial work, I will remove all instances